src/Feature/SignatureHelp/Feature_SignatureHelp.rei

```diff
@@ -0,0 +1,77 @@
+open Oni_Core;
+
+[@deriving show({with_path: false})]
+type provider = {
+  handle: int,
+  selector: list(Exthost.DocumentFilter.t),
+  metadata: Exthost.SignatureHelp.ProviderMetadata.t,
+};
+
+type model;
+
+let initial: model;
+
+[@deriving show({with_path: false})]
+type command =
+  | Show
+  | IncrementSignature
+  | DecrementSignature;
+
+[@deriving show({with_path: false})]
+type msg =
+  | Command(command)
+  | ProviderRegistered(provider)
+  | KeyPressed(option(string), bool)
+  | InfoReceived({
+      signatures: list(Exthost.SignatureHelp.Signature.t),
+      activeSignature: int,
+      activeParameter: int,
+      requestID: int,
+    })
+  | EmptyInfoReceived(int)
+  | RequestFailed(string)
+  | SignatureIncremented
+  | SignatureDecremented;
```

Comment:
Might be worth renaming this such that they describe the event that occurs - like `SignatureIncrementClicked`/`SignatureDecrementClicked`. That way, if there ends up being some subtle difference in behavior between clicking and the command - it's obvious where to handle it.

Reply:
Ah you're right. I changed it in the latest commit. Thanks!